src/components/SearchModal/CurrencySearch.tsx

```diff
@@ -99,18 +101,26 @@ export function CurrencySearch({
   }, [chainId, data?.portfolios])
 
   const sortedTokens: Token[] = useMemo(() => {
-    const portfolioTokens = data?.portfolios?.[0].tokenBalances
-      ?.map((tokenBalance) => {
+    const portfolioTokenBalances = data?.portfolios?.[0].tokenBalances as TokenBalance[] | undefined
+    const portfolioTokens = splitHiddenTokens(portfolioTokenBalances ?? [])
```

Comment:
we might want to hide small balances here - it would match the logic in the mini portfolio

Reply:
the default param is true for checking for small balances